lib/ui/shared/dialogs/schedule_dialog.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:flutter/material.dart';
+import 'package:hng/ui/shared/colors.dart';
+import 'package:hng/ui/view/dm_user/dm_user_viewmodel.dart';
+import 'package:stacked_services/stacked_services.dart';
+
+class ScheduleMess extends StatelessWidget {
+  final DialogRequest request;
+  final Function(DialogResponse) completer;
+
+  const ScheduleMess({Key? key, required this.request, required this.completer})
+      : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
```

Comment:
`Widget build(BuildContext context, DmUserViewModel model) {}`

Now you can just use the `model.` anywhere in this file.